### PR TITLE
Issue 2 view their entries

### PR DESCRIPTION
**Pull request by Samuel-Cho**
Screen Shot 2021-02-10 at 5 07 34 PM

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -24,6 +24,97 @@ function saveNewEntry(event) {
...
+  var divEntryParagraph = document.createElement('div');
+  divEntryParagraph.setAttribute('class', 'entry-paragraph');
+  divEntryTextContainer.appendChild(divEntryParagraph);
+
+  var paragraphEntry = document.createElement('p');
+  var paragraphText = document.createTextNode(entry.note);
+  paragraphEntry.appendChild(paragraphText);
+  divEntryParagraph.appendChild(paragraphEntry);
+
+  $ulEntries.prepend(liJournalEntry);
```
Comment (codingcodymiller) [inline js/main.js]: You should not be appending in this function, you should be returning the `li` that holds the DOM tree for the entry

File: js/main.js [hunk 2]
```diff
@@ -24,6 +24,97 @@ function saveNewEntry(event) {
...
+}
+
+function loadEntries(event) {
+  for (var i = data.entries.length - 1; i >= 0; i--) {
+    createEntry(data.entries[i]);
```
Comment (codingcodymiller) [inline js/main.js]: You should be appending here using the `li` returned from the `createEntry` function

File: js/main.js [hunk 3]
```diff
@@ -24,6 +24,97 @@ function saveNewEntry(event) {
   data.entries.unshift(entryInputs);
   $newEntryForm.reset();
   $urlImage.setAttribute('src', 'images/placeholder-image-square.jpg');
+  createEntry(data.entries[0]);
+  viewEntries();
```
Comment (codingcodymiller) [inline js/main.js]: This will need to be revised after you revise your `createEntry` function.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my notes

**Comment by Samuel-Cho** [comment]
Screen Shot 2021-02-10 at 8 32 27 PM
Screen Shot 2021-02-10 at 8 32 38 PM
Screen Shot 2021-02-10 at 8 32 54 PM

**Review by codingcodymiller — APPROVED** [review]
🎉 🎉 🎉

**Pull request merged**